src/util/net/fd_eth.h

```diff
@@ -283,6 +283,16 @@ fd_vlan_tag( void * _tag,
   return tag;
 }
 
+/* fd_cstr_to_mac_addr parses a MAC address matching format
+   FD_ETH_MAC_FMT from the given cstr and stores the result into mac.
+   On success returns mac.  On failure, returns NULL and leaves mac
+   in an undefined state.  Reasons for failure include that strlen
+   of s is not exactly 17 or that s is not formatted correctly. */
+
+uchar *
+fd_cstr_to_mac_addr( char const * s,
```

Comment:
It's more test-only code going into a runtime lib. Kinda wish we had a better place to put things like this.